_docs/_latest/configure/scanner/policy-violations.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Scanner Violations
```

Comment:
This title and filename could be improved.

Reply:
Updated to rule-violations and Rule Violations